proto/cosmos/bank/v1beta1/tx.proto

```diff
@@ -101,6 +101,7 @@ message MsgUpdateParamsResponse {}
 // message are left unchanged.
 //
 // Since: cosmos-sdk 0.47
+// Note(Heimdall-v2): Not used in Heimdall since multiple denoms are not supported
```

Comment:
I found `// Note(Heimdall-v2):` to be more clear for other devs than `// HV2 `. I can update if you think that's more appropriate.

Reply:
The `Note` is clearer for sure. 
I'd prefer `HV2` just because it's a substring of `TODO HV2` so with one search we can cover both when needed.
So yeah if not a big effort, let's please use `HV2` everywhere. 
Thanks.

Reply:
Okay boss! 🙇